experimental/orchestrator/csharp_dotnetcore/03.declarative-bot/README.md

```diff
@@ -21,7 +21,7 @@ This sample **requires** prerequisites in order to run.
     ```
     - Install bf orchestrator
     ```bash
-    > bf plugins:install @microsoft/bf-orchestrator-cli@beta
+    > bf plugins:install @microsoft/bf-orchestrator-cli@latest
```

Comment:
Not needed, latest deployment will have beta tag.

Reply:
reverted